### DISCUSSION TITLE
Images captured by CARLA camera sensor contains blurred lines

### DISCUSSION THREAD

**Discussion by ksekara** [carla-simulator/carla#6394] [Q&A] [answered]
I'm trying to capture images with following Python code. But the out put is not what I expected.
`camera_bp = bp_lib.find('sensor.camera.rgb') `
`camera_init_trans = carla.Transform(carla.Location(z=2))`
`camera = world.spawn_actor(camera_bp, camera_init_trans, attach_to=vehicle)`
`camera.listen(lambda image: image.save_to_disk('out/%06d.png' % image.frame))`
I have attached sample image below
![001417](https://user-images.githubusercontent.com/69759530/230897278-5ff3ae1a-58e4-4a6d-82d3-abde9adedb0c.png)

**Comment by sjtuyolee** [comment]
I have the same problem

**Reply by xfqbuaa** [reply]
Because you set a wrong position of camera in vehicle inside and get the picture.
Get the sample code in the tutorial  
https://carla.readthedocs.io/en/latest/tuto_G_retrieve_data/#rgb-camera

**Comment by ksekara** [comment, marked as answer]
Able to solve issue by adding following lines
`camera_bp.set_attribute('image_size_x', '640')`
`camera_bp.set_attribute('image_size_y', '480')`
`camera_bp.set_attribute('shutter_speed', '200')`
